nservicebus/upgrades/6to7/index.md

```diff
@@ -161,6 +161,17 @@ The [.Retries message receiver](/nservicebus/recoverability/configure-delayed-re
 
 The [MSMQ transport](/transports/msmq) is no longer part of the NServiceBus NuGet package. It has been moved into a separate package, [NServiceBus.Transport.Msmq](https://www.nuget.org/packages/NServiceBus.Transport.Msmq/).
 
+### Provision of PowerShell Scripts
+
+To facilitate the queue creation for endpoints during deployment, the NuGet package now comes packaged with a `Scripts` folder with two new scripts called, `CreateQueues.ps1` and `DeleteQueues.ps1`. These can be located as part of the package. For example, under `C:\Users\username\.nuget\packages\nservicebus.transport.msmq\1.0.0\lib\net452\Scripts`. 
```

Comment:
As user folders can be stored somewhere else why don't we use the user folder environment variable?

    %USERPROFILE%\.nuget\packages\nservicebus.transport.msmq\1.0.0\lib\net452\Scripts

Reply:
@ramonsmits - good idea.